<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LobbyFrame</class>
 <widget class="QMainWindow" name="LobbyFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="lobbyBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>660</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Game lobby</string>
    </property>
    <widget class="QLabel" name="playersLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>72</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Players:</string>
     </property>
    </widget>
    <widget class="QLabel" name="chatLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>72</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Chat:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="messages">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>641</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="players">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>641</width>
       <height>171</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Nickname</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IP</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Team</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="3" column="2">
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="disconnectButton">
     <property name="geometry">
      <rect>
       <x>525</x>
       <y>10</y>
       <width>125</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>641</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="sendMessageGrid">
      <item row="0" column="1">
       <widget class="QLineEdit" name="messageInput"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="messageLabel">
        <property name="text">
         <string>Message</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="sendMessageButton">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>playersLabel</zorder>
    <zorder>chatLabel</zorder>
    <zorder>messages</zorder>
    <zorder>players</zorder>
    <zorder>disconnectButton</zorder>
   </widget>
   <widget class="QGroupBox" name="conectionBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>TCP/IP connection</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>641</width>
       <height>60</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="inputsGrid">
      <item row="0" column="0">
       <widget class="QLabel" name="nicknameLabel">
        <property name="text">
         <string>Nickname</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nicknameInput"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ipLabel">
        <property name="text">
         <string>Host IP</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="ipInput">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>641</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="buttonsGrid">
      <item row="0" column="0">
       <widget class="QPushButton" name="joinHostButton">
        <property name="text">
         <string>Join</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="makeHostButton">
        <property name="text">
         <string>Become a host</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
